src/transformers/onnx/features.py

```diff
@@ -277,45 +277,68 @@ def feature_to_task(feature: str) -> str:
         return feature.replace("-with-past", "")
 
     @staticmethod
-    def get_model_class_for_feature(feature: str) -> Type:
+    def _validate_framework_choice(framework: str):
+        """
+        Validates if the framework requested for the export is both correct and available, otherwise throws an
+        exception.
+        """
+        if framework not in ["pt", "tf"]:
+            raise ValueError(
+                f"Only two frameworks are supported for ONNX export: pt or tf, but {framework} was provided."
+            )
+        elif framework == "pt" and not is_torch_available():
+            raise RuntimeError("Cannot export model to ONNX using PyTorch because no PyTorch package was found.")
+        elif framework == "tf" and not is_tf_available():
+            raise RuntimeError("Cannot export model to ONNX using TensorFlow because no TensorFlow package was found.")
+
+    @staticmethod
+    def get_model_class_for_feature(feature: str, framework: str = "pt") -> Type:
         """
         Attempt to retrieve an AutoModel class from a feature name.
 
         Args:
             feature: The feature required.
+            framework: The framework to use for the export.
```

Comment:
Nit: since we're updating the docstring, what do you think about adding the types for consistency with the `transformers` conventions? For example, adding:

```
feature (`str`):
    The feature required.
framework (`str`, *optional*, defaults to `pt`):
    The framework to use for the export.
```

Reply:
Done!